packages/orbit-components/src/List/ListContext.ts

```diff
@@ -1,18 +0,0 @@
-import * as React from "react";
-
-import type { Size } from "../common/types";
-import type { Type } from "./types";
-
-interface ListContextProps {
-  readonly size?: Size | null;
-  readonly type?: Type | null;
-}
-
-const ListContext = React.createContext<ListContextProps>({
```

Comment:
Please make sure that this removal won't be a breaking change :)

Reply:
it isn't, I checked with Daniel